src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -420,7 +423,7 @@ bool HolonomicDrive::AtTarget()
 }
 void HolonomicDrive::DriveToFieldElement(double forward, double strafe, double rot, ChassisOptionEnums::DriveStateType driveState)
 {
-    if ((abs(forward) < 0.35 && abs(strafe) < 0.35 && abs(rot) < 0.35) || m_resetPathplannerTrajectory == false)
+    if ((abs(forward) < 0.35 && abs(strafe) < 0.35 && abs(rot) < 0.35) || !m_resetPathplannerTrajectory)
```

Comment:
Should probably just remove this " || !m_resetPathplannerTrajectory" since we aren't a creating trajectories anymore. Can get pushed in, but maybe tonight we clean it up